.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+ATF Test Scripts to check #[issue number]
+
+This PR is **[ready / not ready]** for review.
+
+### Summary
+[Summary of PR changes]
+
+### ATF version
+[ATF version to use]
+
+### Changelog
+##### Breaking Changes
+* [Breaking change info]
+
+##### Enhancements
+* [Enhancement info]
+
+##### Bug Fixes
+* [Bug Fix Info]
+
+### Tasks Remaining:
```

Comment:
Maybe "Tasks Remaining" need to rename ,better to use "Dependencies" to paste info about related tasks.

Reply:
@GetmanetsIrina Renamed in d1e0828